pom.xml

```diff
@@ -0,0 +1,101 @@
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+    implied.
+
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <!-- We could use this parent if we want <parent> <groupId>org.sonatype.oss</groupId> 
+        <artifactId>oss-parent</artifactId> <version>9</version> </parent> -->
+
+    <groupId>io.microprofile</groupId>
+    <artifactId>microprofile-bom</artifactId>
+    <version>1.0.0</version>
+    <packaging>pom</packaging>
+    <name>MicroProfile Bill of Materials</name>
+    <description>MicroProfile Bill of Materials (BOM)</description>
+    <inceptionYear>2016</inceptionYear>
+    <organization>
+        <name>Microprofile.io</name>
+        <url>http://www.microprofile.io</url>
+    </organization>
+
+    <licenses>
+        <license>
+            <name>The Apache Software License, Version 2.0</name>
+            <distribution>repo</distribution>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.html</url>
+        </license>
+    </licenses>
+
+    <scm>
+        <connection>scm:git:git://github.com/microprofile/microprofile-bom.git</connection>
+        <developerConnection>scm:git:git@github.com:microprofile/microprofile-bom.git</developerConnection>
+        <url>http://github.com/microprofile/microprofile-bom</url>
+        <tag>HEAD</tag>
+    </scm>
+
+    <issueManagement>
+        <system>GitHub</system>
+        <url>https://github.com/microprofile/microprofile-bom/issues</url>
+    </issueManagement>
+    <mailingLists>
+        <mailingList>
+            <name>MicroProfile</name>
+            <subscribe>http://groups.google.com/group/microprofile/subscribe</subscribe>
+            <post>microprofile@googlegroups.com</post>
+        </mailingList>
+    </mailingLists>
+
+    <properties>
+        <cdi.version>1.1</cdi.version>
+        <javax.inject.version>1</javax.inject.version>
+        <javax.json.version>1.0</javax.json.version>
+        <javax.jaxrs.version>2.0</javax.jaxrs.version>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>javax.enterprise</groupId>
+                <artifactId>cdi-api</artifactId>
+                <version>${cdi.version}</version>
+                <scope>compile</scope>
+            </dependency>
+            <dependency>

```

Comment:
MicroProfile doesn't define a version for javax.inject, it's derived from what we do depend on, namely cdi-api.

I don't see the versions ever diverging, but to me there's no need for that dependency to be present